docs/server/commands.rst

```diff
@@ -401,17 +402,46 @@ update_tmserver
 
 .. versionadded:: 2.7
 
+.. versionchanged:: 2.7.3 Renamed :option:`--overwrite` to :option:`--refresh`
+   and clarified the behavior of the command.
```

Comment:
*"clarified the behavior of the command"* sounds like avoidable noise. Same for the release notes.